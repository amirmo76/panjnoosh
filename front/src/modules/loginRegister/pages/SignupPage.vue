<template>
  <div class="page-container">
    <div class="image"></div>
    <div class="signup-container">
      <h1>ثبت نام | پنج نوش</h1>
      <SignupForm></SignupForm>
    </div>
  </div>
</template>

<script>
import SignupForm from "../components/SignupForm";

export default {
  name: "SignupPage",
  components: {
    SignupForm
  }
};
</script>

<style scoped>
.page-container {
  background: #1d1d1d;
  display: flex;
  padding: 6rem 10rem;
  flex-direction: row-reverse;
  align-items: center;
}

.signup-container {
  background-color: #151515;
  box-shadow: 0.2rem 1rem 2rem rgba(0, 0, 0, 0.2);
  transform: translateX(3rem);
  width: 70%;
  text-align: right;
  color: #ebebeb;
  padding: 4rem;
}

.image {
  background-image: linear-gradient(to right bottom, #99943d, #99943d),
    url("/assets/images/login.jpg");
  background-blend-mode: darken;
  box-shadow: 0.2rem 1rem 2rem rgba(0, 0, 0, 0.2);
  background-size: cover;
  height: 75rem;
  width: 30%;
}

h1 {
  font-size: 2rem;
  margin-bottom: 3.5rem;
}
</style>

