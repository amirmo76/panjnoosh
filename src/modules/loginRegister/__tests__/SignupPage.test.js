import { shallowMount } from "@vue/test-utils";
import SignupPage from "../pages/SignupPage";

test("should match snapshot", () => {
  const wrapper = shallowMount(SignupPage);
  expect(wrapper.element).toMatchSnapshot();
});
