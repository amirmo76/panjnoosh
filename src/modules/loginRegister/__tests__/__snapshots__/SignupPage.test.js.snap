// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div
  class="page-container"
>
  <div
    class="image"
  />
   
  <div
    class="signup-container"
  >
    <h1>
      ثبت نام | پنج نوش
    </h1>
     
    <signupform-stub />
  </div>
</div>
`;
